include/fmt/color.h

```diff
@@ -399,7 +407,7 @@ template <typename Char> struct ansi_color_escape {
       return;
     }
 
-    for (int i = 0; i < 7; i++) {
+    for (int i = 0; i < 8; i++) {
```

Comment:
I don't understand the buffer logic completely, but you've increased the buffer from a `3 * 4` to a `3 * 8` yet change this loop index from 7 to 8? It might be a good idea to introduce some constants...

Reply:
My bad, thanks!

Reply:
I'll continue tomorrow. Need to sleep. :)